RadarSDK/RadarFraud.h

```diff
@@ -0,0 +1,27 @@
+//
+//  RadarFraud.h
+//  RadarSDK
+//
+//  Copyright © 2021 Radar Labs, Inc. All rights reserved.
+
+#ifndef RadarFraud_h
+#define RadarFraud_h
+#import <Foundation/Foundation.h>
+
+@interface RadarFraud : NSObject
+
+/**
+ `true` if` the user's IP address is being mocked, such as in a simulation. May be `false` if Fraud is not enabled.
```

Comment:
`IP address` should be replaced with `location` here. (and also in `RadarUser`)

Reply:
for consistency, these should match the description strings in `RadarUser`

Reply:
Done.